ShowCase/ShowCase.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
necessary file?

Reply:
Yes, it's the Visual Studio project file used to run the showcase example.